packages/toolpad-app/src/server/auth.ts

```diff
@@ -2,13 +2,20 @@ import express, { Router } from 'express';
 import { Auth } from '@auth/core';
 import GithubProvider, { GitHubEmail, GitHubProfile } from '@auth/core/providers/github';
 import GoogleProvider from '@auth/core/providers/google';
+import AzureADProvider from '@auth/core/providers/azure-ad';
 import { getToken } from '@auth/core/jwt';
 import { AuthConfig, TokenSet } from '@auth/core/types';
 import { OAuthConfig } from '@auth/core/providers';
 import { asyncHandler } from '../utils/express';
 import { adaptRequestFromExpressToFetch } from './httpApiAdapters';
 import { ToolpadProject } from './localMode';
 import * as appDom from '../appDom';
+import { AuthProvider } from '../types';
+
+const SKIP_VERIFICATION_PROVIDERS: AuthProvider[] = [
+  // Azure AD should be fine to skip as the user has to belong to the organization to sign in
+  'azure-ad',
```

Comment:
Also they don't return any data that allows us to determine if emails are verified, from what I searched.